src/components/side-panel/side-panel.component.ts

```diff
@@ -88,53 +90,64 @@ export class SidePanelComponent implements OnInit, OnDestroy {
         return null;
     }
 
-    get hostWidth() {
+    get hostWidth(): string {
         return this.inline ? '100%' : this.cssWidth;
     }
 
-    animationPanelState: SidePanelAnimationState;
+    animationPanelState: SidePanelAnimationState = SidePanelAnimationState.Closed;
+
+    /** Store the current open state */
+    private _isOpen: boolean = false;
 
     protected _onDestroy = new Subject<void>();
 
-    constructor(protected service: SidePanelService, private _elementRef: ElementRef) {}
+    constructor(
+        protected readonly service: SidePanelService,
+        private readonly _elementRef: ElementRef,
+        private readonly _focusOrigin: FocusIndicatorOriginService) { }
 
-    ngOnInit() {
+    ngOnInit(): void {
         this.service.open$.pipe(takeUntil(this._onDestroy)).subscribe(isOpen => {
             this.animationPanelState = isOpen
                 ? this.animate
                     ? SidePanelAnimationState.Open
                     : SidePanelAnimationState.OpenImmediate
                 : SidePanelAnimationState.Closed;
+
+            // only if the open state changed should we emit the latest value
```

Comment:
Should we have `distinctUntilChanged` above to ensure this?

Reply:
Good idea, added in `distinctUntilChanged` operator.